.travis.yml

```diff
@@ -6,6 +6,19 @@ branches:
 
 language: objective-c
 
+env:
+  matrix:
+    - SDK=iphonesimulator8.1 OS=8.1 NAME=iphone 6
+    - SDK=iphonesimulator8.2 OS=8.2 NAME=iphone 6
+    - SDK=iphonesimulator8.3 OS=8.3 NAME=ihpone 6
+    - SDK=iphonesimulator8.4 OS=8.4 NAME=iphone 6
```

Comment:
Travis CI runs them in parallel. Its not too bad: https://travis-ci.org/forcedotcom/SalesforceMobileSDK-iOS/builds/83986417

In any case, if it takes too long then I can reduce it to test only for 8.4 and 9.0